cmd/mt-store-cat/main.go

```diff
@@ -186,7 +188,7 @@ func main() {
 		fmt.Println("Retrieves timeseries data from the cassandra store. Either raw or with minimal processing")
 		fmt.Println()
 		fmt.Println("Usage:")
-		fmt.Printf("	mt-store-cat [flags] <normal|summary> id <metric-id>\n")
+		fmt.Printf("	mt-store-cat [flags] <normal|summary> id <metric-id> <ttl>\n")
```

Comment:
or better, why don't we accept patterns so that people can just the same values that they also use for `ttl` and `agg-settings` in their config.